external-crates/move/crates/move-compiler/tests/move_2024/parser/positional_struct_explicit_type_arg_assign.exp

```diff
@@ -1,3 +1,38 @@
+error[E03003]: unbound module member
+  ┌─ tests/move_2024/parser/positional_struct_explicit_type_arg_assign.move:6:9
+  │
+6 │         Foo <u64>(_) = Foo(0);
+  │         ^^^ Invalid module access. Unbound constant 'Foo' in module '0x42::M'
```

Comment:
This change is very misleading to programmers. We need to figure out how to stop it.